lib/types/multipart.js

```diff
@@ -71,6 +71,8 @@ function Multipart (boy, cfg) {
   let curField
   let finished = false
 
+  this._boundary = boundary
+  this._boundaryFound = false
```

Comment:
```suggestion
  this._boundaryNotFound = true
```

Reply:
Maybe instead of starting with a false value and everyhwere reversing the value with ! maybe use a variable which starts with true and is set to false later...

Only issue: negations are not nice.

Reply:
Your decision